src/qt/walletmodel.cpp

```diff
@@ -190,17 +190,28 @@ WalletModel::SendCoinsReturn WalletModel::prepareTransaction(WalletModelTransact
                 auto pegin_tx = libmw::wallet::CreatePegInTx(m_wallet->GetMWWallet(), rcp.amount);
                 auto commitment = std::vector<uint8_t>(pegin_tx.second.commitment.cbegin(), pegin_tx.second.commitment.cend());
                 transaction.setMapValue("commitment", HexStr(commitment, false));
+                transaction.setMapValue("mweb_recipient", libmw::wallet::GetAddress(m_wallet->GetMWWallet()));
+                transaction.setMapValue("mweb_credit", std::to_string(rcp.amount));
```

Comment:
We also plan to also support pegging-in to someone else's address. That gets us into some complex flows though where we may peg-in X LTC funds, and also include Y mwLTC inputs in the case where you already have some coins pegged-in.

That scenario could really complicate this, so we don't have to implement it now. But maybe just add a TODO describing this scenario?